src/app/app.component.ts

```diff
@@ -136,6 +136,58 @@ export class AppComponent {
       { property: 'og:type', content: 'website' }
     ]);
 
+    this._flags.getFlagsFromService().pipe(
+      take(1),
+      map(flags => {
+        if (flags.enableOneTrust) {
+            window['OptanonWrapper'] = () => {
+                window['OneTrust'].InsertScript("//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit", "body", null, null, "C0003");
+                window['OneTrust'].InsertScript("/assets/js/zendesk.js", "body", null, null, "C0003");
+                setTimeout( () => {
+                    window['zE'](() => {
+                        window['$zopim'](() => {
+                            window['$zopim'].livechat.setOnConnected(() => {
+                                if (this.shouldShowChat(window.location.hash)) {
+                                    window['zE']('webWidget', 'show');
+                                } else {
+                                    window['zE']('webWidget', 'hide');
+                                }
+                            });
+                        });
+                    });
+                }, 1000);
+
+                this._app.googleTranslateLoaded.next(true);
+            }
+        } else {
+            let googleTranslateScript = document.createElement('script');
+            googleTranslateScript.setAttribute('src','//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit');
```

Comment:
Because you've made your own observer with `rxjs`, I think you don't want this anymore? Maybe we weren't ever using it anyway. You could also choose to take advantage of it instead of doing the `rxjs` complexity.

```suggestion
            googleTranslateScript.setAttribute('src','//translate.google.com/translate_a/element.js');
```

Reply:
Changes made in the recent commit `001d1f6`.